simulation_ws/src/turtlebot3_simulation_description/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+################################################################################
```

Comment:
Same question I had at the sync up - why are you replacing turtlebot3_description_reduced_mesh with this? also makes it more difficult to review as you're adding a bunch of gazebo assets that we didn't have before.

Reply:
Thanks for reviewing & share the valuable comments.

Apologize for the inconvenience, As per your suggestion, We have used reduced mesh TurtleBot3 models (i.e. waffle_pi_base_reduced.stl, burger_base_decimated.stl etc) in **turtlebot3_description_reduced_mesh package**, removed "turtlebot3_simulation_description" package and urdf files to launch it.